### PR TITLE
Improve test coverage for algorithms in dominating_set.py (PR for issue 6032)

### PR DESCRIPTION
**Pull request by chimaobi-okite**
added tests to improve test coverage of dominating_set.py

### CODE REVIEW

**Comment by chimaobi-okite** [comment]
please review @MridulS- I updated this to account for tests previously ignored.

File: test.edgelist [hunk 1]
```diff
@@ -0,0 +1,3 @@
+0 1 {}
```
[the remark this answers is not in the document]
Comment (chimaobi-okite) [reply test.edgelist]: I am very sorry about that @MridulS, they have been removed. Please check.

**Comment by chimaobi-okite** [comment]
Hi @MridulS, kindly help me review

**Review by MridulS — APPROVED** [review]
Thanks @chimaobi-okite !

**Pull request merged**